.eslintrc.js

```diff
@@ -6,5 +6,5 @@ module.exports = {
   rules: {
     'no-proto': 0
   },
-  plugins: ['jest']
+  plugins: ['jest'],
```

Comment:
Unnecessary trailing comma found in the plugins array. Please remove the comma to adhere to the ESLint rules.